<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Detalles</class>
 <widget class="QDialog" name="Detalles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detalles</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>50</y>
     <width>401</width>
     <height>341</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../detalles_logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblCamara">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblSensor1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblSensor2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblMotor3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>310</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblMotor2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblMotor1">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>70</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>70</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-weight: bold;
color: #222;
letter-spacing: 1px;</string>
   </property>
   <property name="text">
    <string>Camara 
Camera </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-weight: bold;
color: #222;
letter-spacing: 1px;</string>
   </property>
   <property name="text">
    <string>Sensor 1 
Sensor 1 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-weight: bold;
color: #222;
letter-spacing: 1px;</string>
   </property>
   <property name="text">
    <string>Sensor 2 
Sensor 2 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-weight: bold;
color: #222;
letter-spacing: 1px;</string>
   </property>
   <property name="text">
    <string>Motor 1 
Motor 1 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>190</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-weight: bold;
color: #222;
letter-spacing: 1px;</string>
   </property>
   <property name="text">
    <string>Motor 2 
Motor 2 </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>300</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;
font-weight: bold;
color: #222;
letter-spacing: 1px;</string>
   </property>
   <property name="text">
    <string>Motor 3 
Motor 3 </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVolver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>OpenHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2176ae;
color: white;
font-weight: bold;
font-size: 16px;
border-radius: 8px;
padding: 8px 28px;
border: none;
text-align: left;</string>
   </property>
   <property name="text">
    <string>VOLVER 
BACK</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
